i18n/en.json

```diff
@@ -2800,7 +2800,7 @@
   },
   {
     "id": "app.import.validate_user_import_data.notify_props_comments_trigger_invalid.error",
-    "translation": ""
+    "translation": "Invalid Comments Trigger Prop value for user."
```

Comment:
Do you mean the `Comments` field from https://docs.mattermost.com/deployment/bulk-loading.html#fields-of-the-usernotifyprops-object?

Reply:
Yes, you are right, in the code it is called CommentsTrigger, but it refers to comments.